lib/t/accounts.ex

```diff
@@ -547,6 +597,18 @@ defmodule T.Accounts do
     |> Repo.all()
   end
 
+  def get_feed_filter(user_id) do
```

Comment:
what's the point of this function? where is it called? why can't profile in memory be used to construct feed_filters?

Reply:
It fetches feed_filers for feedChannel: https://github.com/getsince/test3/blob/8d4c6eb69d046f136ba2ddada861b46ccc4e224c/lib/t_web/channels/feed_channel.ex#L16

As socket assigns for different channels are separate, feedChannel doesn't have profile from memory. Alternatively the whole profile + gender_preferences could be assigned to feedChannel's socket

Reply:
if it's only for feed, than it should be in `Feeds` context

Reply:
`Accounts` are for working with users, profiles, auth, etc.